Code/Legacy/CrySystem/XML/xml.cpp

```diff
@@ -1247,9 +1247,7 @@ bool CXmlNode::saveToFile(const char* fileName)
 
 bool CXmlNode::saveToFile([[maybe_unused]] const char* fileName, size_t chunkSize, AZ::IO::HandleType fileHandle)
 {
-#ifdef WIN32
-    CrySetFileAttributes(fileName, 0x00000080);  // FILE_ATTRIBUTE_NORMAL
-#endif //WIN32
+    CrySetFileAttributes(fileName, FILE_ATTRIBUTE_NORMAL);
```

Comment:
Are you sure that we should remove the #ifdef WIN32 here?

Reply:
CrySetFileAttributes is implemented for all platforms though?
Even if Linux version is just a placeholder :smile:

Reply:
Yeah just not sure if there is/was a reason that it was only being done on Windows. Should be fine, just brought it up because it's a potential change in behavior on platforms that don't get as much test/usage coverage as Windows.